.github/workflows/CI.yml

```diff
@@ -141,10 +140,12 @@ jobs:
         run: |
           PYTEST_ARGS+=" --ignore=openforcefield/tests/test_examples.py"
           PYTEST_ARGS+=" --ignore=openforcefield/tests/test_links.py"
+          # if [[ "$GITHUB_EVENT_NAME" == "schedule" ]]; then
           if [[ "$RDKIT" == true && "$OPENEYE" == true ]]; then
-            PYTEST_ARGS+="--doctest-modules"
+            # Switch back to cron-only before merging
+            PYTEST_ARGS+=" --runslow"
           fi
-          pytest $PYTEST_ARGS $PACKAGE/
+          pytest $PYTEST_ARGS openforcefield/tests/
```

Comment:
There are two changes here, which get mushed together in the history

1. Drop `--doctest-mocules` for now, as it's buggy and/or giving us problems that I'm pushing out of the scope of this PR.
2. Adding `--runslow` to the cron job, although right now it's triggered in the TRUE/TRUE case so it runs on commits.